packages/truffle-debugger/lib/evm/sagas/index.js

```diff
@@ -45,14 +56,33 @@ export function* begin({ address, binary }) {
 }
 
 export function* callstackSaga () {
+  let contexts;
+  let instances;
+
   while (true) {
     yield take(TICK);
-    debug("got TICK");
+
+    // HACK contexts and instances won't be available until after first TICK
```

Comment:
I don't understand the point of this block; I thought we just got a `TICK`?  What's going on here?